pom.xml

```diff
@@ -9,59 +9,53 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <mainClass>uk.gov.pay.ledger.app.LedgerApp</mainClass>
-
-        <dropwizard.version>2.1.12</dropwizard.version>
-        <jackson.version>2.16.1</jackson.version>
-        <testcontainers.version>1.19.5</testcontainers.version>
-        <postgresql.version>42.7.2</postgresql.version>
         <pay-java-commons.version>1.0.20240205110547</pay-java-commons.version>
-        <junit5.version>5.10.2</junit5.version>
         <prometheus.version>0.16.0</prometheus.version>
         <surefire.version>3.2.5</surefire.version>
-        <guice.version>5.1.0</guice.version>
-        <rest-assured.version>5.4.0</rest-assured.version>
-        <swaggger-version>2.2.20</swaggger-version>
+        <swaggger.version>2.2.20</swaggger.version>
```

Comment:
I wonder if now might be the time to remove the extraneous ‘g’.

Reply:
👍